src/i18n/en-US/index.js

```diff
@@ -1,9 +1,15 @@
 export default {
-  project: {
```

Comment:
Commit should be put after [Setup test environment for Vue components](https://github.com/ditrit/leto-modelizer/pull/25/commits/f5e2f23d9465b212396c723a2a6d81716d998daa)